### DISCUSSION TITLE
Failed : GCP Wordpress Multisite Configuration

### DISCUSSION THREAD

**Discussion by DipendraKSahay** [evertramos/docker-wordpress#60] [General]
Hello Experts!

I am still in the learning process therefore I always face many challenges. I have a website created on GCP with WordPress Multisite. Last week, it was down which forced me to recreate the instances and design all layouts on WP Multisite. Now, I have a serious problem. I am unable to find wpconfig.php to edit the website address. 

Previously, it used to be /opt/bitnami/apps/wordpress/wpconfig.php

I used SSH to enter root [ sudo -i ] and tried to run command [ nano /opt/bitnami/apps/wordpress/wpconfig.php ]

Can you experts help me? please!

**Comment by evertramos** [comment]
Hello @DipendraKSahay,

This setup does not use the bitnami version, so the files will be located in the /var/www/html, as mounted in the container:

https://github.com/evertramos/docker-wordpress/blob/e418130bae3b8ffc295040bd63a90afbf54af0fe/docker-compose.yml#L23

**Reply by DipendraKSahay** [reply]
Hello @evertramos 

I still could not trace any path to modify Bitnami multisite under GCP. Even Help & Tutorials are unavailable for the latest modification Bitnami has made as well of GCP. 

Is there a quick reference to point my physical web address to Bitnami Multisite on GCP? My Cloud DNS is configured correctly, the only I need to change is in wpconfig.php file.

I shall be highly obliged to you if you can help. Please!

**Reply by evertramos** [reply]
@DipendraKSahay your wp-config.php is in the server in the SITE_FILES path configured in your `.env` file, where you clone this repo. Otherwise you can check the container running the `docker inspect CONTAINER_NAME` (check your container name) so you will see where the wp-config.php are. You could even get in the container by `docker exec -it CONTAINER_NAME bash` and just list the files (`ls -la | grep wp-config.php`) and you will find it.

**Reply by evertramos** [reply]
You might reach me by e-mail for further information. (https://github.com/evertramos)
